client/src/components/Admin/UsersList.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import { Card, CardActions, CardHeader, CardText } from 'material-ui/Card';
+import FlatButton from 'material-ui/FlatButton';
+
+
+const UsersList = ({ user, deleteUser }) => {
+  const ondeleteUser = () => {
+    deleteUser(user.id)
+      .catch(error => console.log(error));
+  };
+
+  return (
+    <ul className="collection">
+      <li className="collection-item avatar" key={user.id}>
+        <img src="images/yuna.jpg" alt="" className="circle" />
+        <span className="title">{user.username}</span>
+        <p>Role: {user.roleTitle} </p>
+        <p>Date Joined: {user.roleTitle} </p>
+        <a className="secondary-content" onClick={ondeleteUser}>
+          <i className="close waves-effect waves-light material-icons">close</i></a>
```

Comment:
This and the next block should be indented properly i.e should be 
```
<a className="secondary-content" onClick={ondeleteUser}>
   <i className="close waves-effect waves-light material-icons">close</i>
</a>
```
 Also please change everywhere you have similar code